.changeset/violet-points-sparkle.md

```diff
@@ -0,0 +1,13 @@
+---
+'@keystonejs/server-side-graphql-client': minor
+---
+
+Enhanced the `getItems` functionality by adding: 
+- `sortBy`
+- `first`
+- `skip`
+
+These variable are part of the public interface, and allow users to achieve some advance use cases like sorting and skipping.
+
+**Note**: We are still keeping the internal pagination behaviour as it is, but now prioritising `first` over the `pageSize`.
```

Comment:
I think we can remove this note, nothing in the described changes implies that pagination changed, so it's more confusing to call it out than to not.